tiled/client/context.py

```diff
@@ -661,6 +661,11 @@ def configure_auth(self, tokens, remember_me=True):
         auth.sync_set_token("refresh_token", tokens["refresh_token"])
         self.http_client.auth = auth
 
+    @property
+    def authenticated(self):
```

Comment:
I'm trying to form the habit of including type hints on all new Tiled code, as we gradually type the codebase.

```suggestion
    def authenticated(self) -> bool:
```

Also, would you add a docstring to this property? Just a one-liner is sufficient.